packages/create-vite/template-react/src/style.css

```diff
@@ -0,0 +1,35 @@
+:root {
```

Comment:
Other templates are using `index.css`, so it might be better to change this to `index.css`.